### ISSUE TITLE
[FRG] Alignment on Print elements does not align within assigned box

### ISSUE THREAD

**Issue by beauwest** [NathanaelA/fluentreports#228]
Consider this element definition
```
{"text":"Welcome to FluentReports","type":"print","settings":{"align":"center","width":234,"absoluteX":234,"absoluteY":0}}
```

Steps to generate:
1. Create a new Print element & change the text
1. Set the width to 234
1. Set the X to 234
1. Change the alignment to "Center"

The expectation, is that I would have a print that is half the width of my page, aligned to the right-hand side of my page. But then the text would be centered within the right-hand "column" on my page.

The result:
![Screen Shot 2020-10-08 at 12 02 01 PM](https://user-images.githubusercontent.com/193887/95490634-180ccd00-095e-11eb-9a29-87d586aab2a0.png)

**Comment by NathanaelA** [comment]
This is fixed in the next version.     The Text Box (i.e. 234 pixels text width/wrapping area) is moved to the proper alignment, and the text inside the text width/wrapping area is aligned to the proper alignment.    (Demo34 shows the output of a fixed width text box with alignments)

**Comment by NathanaelA** [comment]
Fixed in 1.3.8
